src/scss/base.scss

```diff
@@ -1,15 +1,16 @@
+
 @include oFontsInclude(BentonSans, normal);
 
-.o-ft-header {
+.o-header {
 	// 80 px for the primary container and 5 for an empty secondary or 25 for a secondary with content
 	min-height: 85px;
 	max-height: 125px;
 	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

Comment:
`font-family: oFontsGetFamily(use-case-name);`

Available use cases:

* header (eg h1, h2, h3 etc)
* body (all other text not in interaction/UI elements)
* ui (for interaction elements: buttons, forms etc)
* small-caps 
* editorial-headline (editorial means words written by a journalist).
* editorial-body (the main copy of an FT article)
* editorial-aside (for text that's part of the editorial copy but not the main content: figures, asides, big numbers,  picture captions, graphic furniture).

These pretty much represent the uses I've observed.